.changeset/lemon-goats-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-apache-airflow': patch
```

Comment:
change this to `minor` as adding new required types to the `ApacheAirflowApi` is a breaking change for anyone that would implement their own.

Reply:
Can I just change this or do I have to run `yarn changeset` again?